frontend/src/components/PetProfile/PetProfile.jsx

```diff
@@ -0,0 +1,70 @@
+import { useSelector } from 'react-redux'
+import './PetProfile.css'
+import { Link, useParams } from 'react-router-dom'
+
+const PetProfile = () => {
+    const { petId } = useParams()
+    const pet = useSelector(selectPet(parseInt(petId)))
+    const petAttributes = Object.entries(pet)
+
+    const renderAttributes = () => {
+        return petAttributes.map( (pair, idx) => {
+            let key = pair[0]
+            let val = pair[1]
```

Comment:
`return petAttributes.map( ([key, val], idx) => {`
instead, perhaps?

Reply:
ah! that's what i was trying to do. ty!